charts/rancher-backup/templates/network-policy-allow-all.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: networking.k8s.io/v1
```

Comment:
same comment as https://github.com/rancher/charts/pull/1155#discussion_r623435213. Let's standardize this

Reply:
sovled.